python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -521,18 +699,32 @@ def run_cutouts(args):
     )
 
     butler = lsst.daf.butler.Butler(args.repo, collections=args.collections)
-    data = select_sources(butler, args.instrument, args.n,
-                          sqlitefile=args.sqlitefile,
-                          postgres_url=args.postgres_url,
-                          namespace=args.namespace)
+    apdb_query = _make_apdbQuery(butler,
+                                 args.instrument,
+                                 sqlitefile=args.sqlitefile,
+                                 postgres_url=args.postgres_url,
+                                 namespace=args.namespace)
+    data = select_sources(apdb_query, args.limit)
 
     config = ZooniverseCutoutsConfig()
     if args.configFile is not None:
         config.load(os.path.expanduser(args.configFile))
-    config.validate()
```

Comment:
Just checking that this removal was deliberate, given you're still calling `freeze`

Reply:
Yes, DM-971 made configs validate on Task init, but DM-972 is needed to get them to be frozen (that's going to be trickier, due to bad usages in our tests).